app/assets/stylesheets/style/btn.scss

```diff
@@ -1,5 +1,72 @@
 .btn {
+  border-radius: 2px;
```

Comment:
Properties should be ordered border, border-radius, box-shadow, color, cursor, display, font-size, padding, position, text-decoration